user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -19,111 +19,295 @@ redirect_from:
+to take longer than an installation on a internal storage disk.
+
+<div class="alert alert-info" role="alert">
+    <i class="fa fa-question-circle"></i>
+    <b>Note : </b> See <a href="/doc/custom-install/">the Custom Installation</a> for more options. 
+</div>
+
+You are now ready to go. Press the "Begin Installation" button.
+
+<img src="/attachment/wiki/InstallationGuide/installation-summary-ready.png">
+
+<h3>Pick your user name</h3>
+
+While the installation is ongoing, a new user needs to be created. Click on "User Creation" to define a new user with administrator privileges and a password. 
+Just as for the disk encryption, this password should be complex. The root account is deactivated and should remain as such.
+
+<img src="/attachment/wiki/InstallationGuide/account-name-and-password.png">
+
+When the installation is complete, click on the "Reboot" button. Don't forget to remove the installation media, otherwise you may end up seeing the Qubes OS boot screen again. 
+
+<div class="alert alert-info" role="alert">
+    <i class="fa fa-question-circle"></i>
+    <b>Note : </b> By design, Qubes OS is a single user operating system. 
+</div>
+
+Post-installation
+-----------------
+
+<h3>First boot</h3>
+
+If Qubes OS has been successfully installed, you should see the GRUB menu during the booting process.
+
+<img src="/attachment/wiki/InstallationGuide/grub-boot-menu.png">
+
+Just after this screen, you will be asked to unlock your storage device. 
+
+<img src="/attachment/wiki/InstallationGuide/unlock-storage-device-screen.png">
+
+<h3>Initial Setup</h3>
+
+You're almost done. Before you can start using Qubes OS, some configuration is needed. By default, Qubes OS will create a number of qubes, based 
+on Fedora templates or Whonix templates, so that you can have a more ready-to-use environnement from the get-go.
+
+<img src="/attachment/wiki/InstallationGuide/initial-setup-menu.png">
+
+* **Create default system qubes** : it is recommended to use system qubes as they offer some of the core functionalities brought by Qubes OS, including network isolation and disposable qubes  
+* **Create default application qubes** : application qubes are pre-configured qubes meant to be used for specific purposes/categories, such as work or personal. 
+* **Create Whonix Gateway and Workstation qubes** : in order to be able to use Tor for dedicated qubes, you need this option to be activated. 
+    * **Enabling system and template updates over the Tor anonymity network using Whonix** : this option allows the use of Tor system-wide rather than only for specific qubes.  
+* **Create USB qube holding all USB controllers** : just like the network qube for the network stack, the USB qube allows to capture the USB controller and to manage USB devices through it. 
+    * Use sys-net qube for both networking and USB devices : **???**
```

Comment:
This combines USB qube functionality into sys-net qube. It saves some memory (only sys-net running, instead of sys-net + sys-usb), but also allows to easily use USB networking devices (like 3G/LTE modems) directly in sys-net.

Reply:
Thanks for the clarification